### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by openpaul** [Free-Software-for-Android/Standalone-Calendar#9]
I was just wondering if you are still maintaining this app?

**Comment by dschuermann** [comment]
nope, I never maintained it :) I just created some build files to build the AOSP Calendar. If you want to take over maintainig feel free to fork

**Comment by mathstuf** [comment]
Yeah, I can see this app becoming a lot more popular with the lollipop version being a downer (so far), so +1 for resurrecting it (somewhere). I, unfortunately, don't have the knowhow in java nor the time :( . I'm willing to test though.

**Comment by thewhitetulip** [comment]
I would like to convert the cm12 calendar into a standalone calendar, how can I update this to the cm-12 version and not break backward compatibility?

**Comment by 459below** [comment]
I looked at what Schürmann did and replicated those steps. Though I didn't use any Cyanogen repos, but only AOSP repos directly. Thus I couldn't fork this project. One could build a 6.0 Version of the App using eclipse and those steps I documented in the README.md

https://github.com/459below/platform_packages_apps_calendar/tree/standalone

But note, that I won't give any additional help.

**Comment by thewhitetulip** [comment]
Hey, actually we do not need a stand alone calendar app, if you search in the Fdroid then you can find the app which uses a clever hack to use the existing calendar without the need to sync, it adds a local account, I forgot the name, but if you search for calendar you'll find it, I stopped using android phone so unable to tell the exact name

**Comment by dschuermann** [comment]
This app is maintained by me and called Offline Calendar ;)

**Comment by 459below** [comment]
@thewhitetulip I don't think the purpose of Offline Calendar and Standalone Calendar are the same. As it says in the description "Some people do not have the open source calendar from AOSP and are forced to use either the proprietary Google Calendar from Google Play or the shipped crippled calendar from e.g. Samsung." So this Standalone Calendar solves the issue of the blocked package name "com.android.calendar" on e.g. Samsung stock roms. So you would be forced to use S-Planner or Google Calendar. Standalone Calendar would be the only way for one to use the own e.g. ownCloud Calendar with a AOSP like calendar properly, since Google Calendar seems to always prefer Google's Calendar as destination for new events.

@dschuermann I updated the description to give credit to you. I did just copy your steps after all.
